chore: add common env vars for junit and json reporters (#30611)
microsoft/playwright: docs/src/test-reporters-js.md

| Environment Variable Name | Reporter Config Option| Description | Default
|---|---|---|---|

> | `PLAYWRIGHT_JSON_OUTPUT_DIR` | | Directory to save the output file. Ignored if output file is not specified. | Config directory or `cwd`.

Reviewer: ```suggestion
| `PLAYWRIGHT_JSON_OUTPUT_DIR` | | Directory to save the output file. Ignored if output file is specified. | `cwd` or config directory.
```

> | `PLAYWRIGHT_JSON_OUTPUT_DIR` | | Directory to save the output file. Ignored if output file is not specified. | Config directory or `cwd`.

Reviewer 2: ```suggestion
| `PLAYWRIGHT_JSON_OUTPUT_DIR` | | Directory to save the output file, used together with `PLAYWRIGHT_JSON_OUTPUT_NAME`. | JSON report is printed to stdout.
```

> | `PLAYWRIGHT_JSON_OUTPUT_NAME` | `outputFile` | Full path for the output. | JSON report is printed to stdout.

Reviewer: ```suggestion
| `PLAYWRIGHT_JSON_OUTPUT_NAME` | `outputFile` | Base file name for the output, relative to the output dir. | JSON report is printed to the stdout.
```

> | `PLAYWRIGHT_JSON_OUTPUT_NAME` | `outputFile` | Full path for the output. | JSON report is printed to stdout.

Author: Update to "Base name..."

> | `PLAYWRIGHT_JSON_OUTPUT_FILE` | `outputFile` | Full path for the output. If defined, `PLAYWRIGHT_JSON_OUTPUT_DIR` and `PLAYWRIGHT_JSON_OUTPUT_NAME` will be ignored. | JSON report is printed to stdout.

Reviewer: ```suggestion
| `PLAYWRIGHT_JSON_OUTPUT_FILE` | `outputFile` | Full path for the output. If defined, `PLAYWRIGHT_JSON_OUTPUT_DIR` and `PLAYWRIGHT_JSON_OUTPUT_NAME` will be ignored. | JSON report is printed to the stdout.
```

> | `PLAYWRIGHT_JSON_OUTPUT_FILE` | `outputFile` | Full path for the output. If defined, `PLAYWRIGHT_JSON_OUTPUT_DIR` and `PLAYWRIGHT_JSON_OUTPUT_NAME` will be ignored. | JSON report is printed to stdout.

Reviewer 2: ```suggestion
| `PLAYWRIGHT_JSON_OUTPUT_FILE` | `outputFile` | Full path to the output file. If defined, `PLAYWRIGHT_JSON_OUTPUT_DIR` and `PLAYWRIGHT_JSON_OUTPUT_NAME` will be ignored. | JSON report is printed to stdout.
```

| Environment Variable Name | Reporter Config Option| Description | Default
|---|---|---|---|

> | `PLAYWRIGHT_JUNIT_OUTPUT_DIR` | | Directory to save the output file. Ignored if output file is not specified. | Config directory or `cwd`.

Reviewer: ditto for all three, also check blob
